pkg/controller/controller.go

```diff
@@ -11,18 +11,8 @@ func Ping(c *gin.Context) {
 	})
 }
 
-// Temporary for testing
-func ScrapeJobcenter(c *gin.Context) {
-	jobs := scraper.ScrapeJobcenter()
-	c.JSON(200, gin.H{
-		"success": true,
-		"jobs":    jobs,
-	})
-}
-
-// Temporary for testing
-func ScrapeBruneida(c *gin.Context) {
-	jobs := scraper.ScrapeBruneida()
+func GetJobs(c *gin.Context) {
+	jobs := service.GetJobs()
```

Comment:
It seems that the way to handle an error in gin is to check for err then call `panic(err)` if it is not ok and let the middleware handle the logging and reponse

Log.fatal's doc says it will log the error followed by os.Exit(1) which I think is probably not what we want